ci/ReleaseTool/PrepCommand.cs

```diff
@@ -118,38 +101,17 @@ public int Run()
                         gitClient.CheckoutRemoteBranch(options.SourceBranch);
 
                         // 3. Makes repo-specific changes for prepping the release (e.g. updating version files, formatting the CHANGELOG).
-                        switch (options.GitRepoName)
+                        if (Common.UpdateVersionFilesWithEngine(gitClient, gitRepoName, options.Version, "-rc", options.EngineVersions, Logger))
```

Comment:
We only commit & push if there actual changes to any of the files, before Prep would commit regardless (but PrepFull would not, so with this change they are consistent)